static/app/views/aiMonitoring/aiMonitoringDetailsPage.tsx

```diff
@@ -178,7 +179,7 @@ function PageWithProviders({params}: Props) {
   return (
     <ModulePageProviders
       title={[spanDescription ?? t('(no name)'), t('Pipeline Details')].join(' — ')}
-      baseURL="/ai-monitoring/"
+      baseURL={BASE_URL}
```

Comment:
This also includes sentry locale package which you don't need in this file. This could easily come to a point where it effects build time and makes it harder to eliminate dead code. I'm against this pattern.

Reply:
🤔 I see your point, but:

1. In this PR, every file that imports `BASE_URL` already imports `t`, so I'm not introducing a new import
2. Colocating regular constants and translated strings is a common pattern

If you feel strongly that this pattern is bad, I would love to see proof that it has a meaningful impact on build time, an argument that it's worth sacrificing colocation, and some suggestions on what to do instead

I think the FE TSC or the frontend Slack channel are a more appropriate venue for this, since it needs some discussion. If we all agree, we can introduce this to our developer guidelines and work on improving the situation

In the meantime, is this worth blocking this PR for, or is it a nit? If it's the latter, blocking the PR is probably not appropriate, unless I'm misinterpreting our [PR reviewer guidelines](https://develop.sentry.dev/code-review/#guidelines-forreviewers)